frappe/utils/data.py

```diff
@@ -949,6 +950,11 @@ def get_timespan_date_range(
 				get_quarter_start(add_to_date(today, months=3)),
 				get_quarter_ending(add_to_date(today, months=6)),
 			)
+		case "in the past":
```

Comment:
This is a very weird filter and not at all intuitive!

Reply:
What if I want to know what was sold up until yesterday? After all, it would be unreasonable on my part to include today's sales since the day isn't over yet. Anyway, I see several scenarios where the user could use this filter.